client/my-sites/hosting/sftp-card/ssh-key-card.tsx

```diff
@@ -54,7 +54,7 @@ function SshKeyCard( { deleteText, siteId, sshKey }: SshKeyCardProps ) {
 	);
 	const { sha256, user_login, name, attached_at } = sshKey;
 	return (
-		<SSHKeyCard.Root>
+		<SSHKeyCard.Root className="ssh-keys-card">
```

Comment:
As we're using CSS-in-JS here, how about extending the base component with more styles instead of using both CSS-in-JS and plain CSS?

Reply:
@zaguiini Good point!

I just tried this refactor:

```diff
﻿﻿diff --git a/client/my-sites/hosting/sftp-card/ssh-key-card.tsx b/client/my-sites/hosting/sftp-card/ssh-key-card.tsx
index 1ee9095b69..fdd57a3f2c 100644
--- a/client/my-sites/hosting/sftp-card/ssh-key-card.tsx
+++ b/client/my-sites/hosting/sftp-card/ssh-key-card.tsx
@@ -1,3 +1,4 @@
+import styled from '@emotion/styled';
 import { sprintf } from '@wordpress/i18n';
 import { useI18n } from '@wordpress/react-i18n';
 import i18n from 'i18n-calypso';
@@ -14,6 +15,16 @@ interface SshKeyCardProps {
 	sshKey: AtomicKey;
 }
 
+const MEDIA_QUERIES = {
+	wide: '@media screen and ( min-width: 1280px )',
+};
+
+const SSHKeyCardRoot = styled( SSHKeyCard.Root )( {
+	[ MEDIA_QUERIES.wide ]: {
+		paddingLeft: '24px',
+	},
+} );
+
 const noticeOptions = {
 	duration: 3000,
 };
@@ -54,7 +65,7 @@ function SshKeyCard( { deleteText, siteId, sshKey }: SshKeyCardProps ) {
 	);
 	const { sha256, user_login, name, attached_at } = sshKey;
 	return (
-		<SSHKeyCard.Root className="ssh-keys-card">
+		<SSHKeyCardRoot>
 			<SSHKeyCard.Details>
 				<SSHKeyCard.KeyName>
 					{ user_login }-{ name }
@@ -80,7 +91,7 @@ function SshKeyCard( { deleteText, siteId, sshKey }: SshKeyCardProps ) {
 			>
 				{ deleteText }
 			</SSHKeyCard.Button>
-		</SSHKeyCard.Root>
+		</SSHKeyCardRoot>
 	);
 }
 
diff --git a/client/my-sites/hosting/sftp-card/ssh-keys.scss b/client/my-sites/hosting/sftp-card/ssh-keys.scss
index 74caec0364..a86cf4afc1 100644
--- a/client/my-sites/hosting/sftp-card/ssh-keys.scss
+++ b/client/my-sites/hosting/sftp-card/ssh-keys.scss
@@ -2,14 +2,6 @@
 	max-width: calc(100vw - 32px);
 }
 
-@include breakpoint-deprecated( ">1280px" ) {
-	.hosting__main-layout-col {
-		.card.ssh-keys-card {
-			padding-left: 24px;
-		}
-	}
-}
-
 .ssh-keys__form {
 	display: flex;
 	flex-direction: column;

```

However, it doesn't appear the style is specific enough:

<img width="1443" alt="image" src="https://user-images.githubusercontent.com/36432/201760387-b7977b2f-a0b9-4c28-946e-1aa0d36c1035.png">

I think we might as well keep things as-is for now.

Reply:
I made a commit that builds on this change but uses the double-ampersand trick to increase specificity.

Reply:
@mpkelly Nice, thanks!